using_openshift/builds.adoc

```diff
@@ -80,28 +80,30 @@ SHA1 tag or a branch name.
 the source code repository, where the build looks for the application source
 code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
+====
 
 [[using-the-sti-environment-file]]
-== STI environment file
+== STI Environment File
 
-STI has a capability to set environment values, specified in the
-.sti/environment file inside your application, those variables will be
-present during the build process and in the final docker image.
+link:../image_writers_guide/sti.html[STI] enables you to set environment values specified in the
+*_.sti/environment_* file inside your application. These variables are present
+during the build process and in the final docker image. The complete list of
+supported environment variables are available in the documentation for each
+image.
```

Comment:
Question about "inside your application". Is "inside" the best word to use? Also, it might need to be clearer about whether it's the file or the envar that's "inside" the app.